tests/torch/test_hybrid_converter.py

```diff
@@ -37,17 +37,9 @@ def run_hybrid_llm_test(
 ):
     """Run the test for any model with its private module names."""
 
-    # Multi-parameter strategy is used in order to speed-up the FHE executions
-    configuration = Configuration(
```

Comment:
same, these options are the QM's default, so let's avoid some confusion / unexpected mismatches